biohansel/cli.py

```diff
@@ -159,7 +163,7 @@ def subtype(scheme,
     input_contigs, input_reads = collect_inputs(**locals())
     if len(input_contigs) == 0 and len(input_reads) == 0:
         no_files_exception = click.UsageError('No input files specified!')
-        click.secho('Please see -h/--help for more info', err=True)
+        logging.info('Please see -h/--help for more info', err=True)
```

Comment:
Please revert this change. If there are issues with the code related to the `subtype` command, let me know in https://github.com/phac-nml/biohansel/pull/52

Reply:
yes this has been addressed in d141a97fd0744cfd2f291e96d02ced7710935307